packages/builder/src/pages/builder/portal/users/users/[userId].svelte

```diff
@@ -164,40 +165,45 @@
     return label
   }
 
-  async function updateUserFirstName(evt) {
+  async function saveUser() {
     try {
-      await users.save({ ...user, firstName: evt.target.value })
+      await users.save({ ...user, ...userFieldsToUpdate })
+      userFieldsToUpdate = {}
       await fetchUser()
     } catch (error) {
       notifications.error("Error updating user")
     }
   }
 
+  async function updateUserFirstName(evt) {
+    userFieldsToUpdate = { ...userFieldsToUpdate, firstName: evt.target.value }
```

Comment:
NAB: we don't need to clone the object here - we can just do `userFieldsToUpdate.firstName = evt.target.value`.
Same applies to the other updates as well.